python/doc/examples/meta_modeling/general_purpose_metamodels/plot_expert_mixture.py

```diff
@@ -3,35 +3,33 @@
 ==================
 """
 # %%
-# In this example we are going to approximate a piece wise continuous function using an expert mixture of metamodels.
+# In this example we are going to approximate a piece-wise continuous function using an expert mixture of metamodels.
 #
-# The metamodels will be represented by the family of :math:`f_k \forall \in [1, N]`:
+# The metamodels will be represented by the family of :math:`f_k \forall k \in [1, N]`:
```

Comment:
The interval $[1, N]$ is a real interval, which is not the interval we want here. We should instead write `[\![1 .. N ]\!]`that is rendered as :

![image](https://github.com/openturns/openturns/assets/31351465/af7a182b-beb0-4b2a-a974-f098c19ffae9)

or $\{1, ..., N\}$ that is a set of integers. I would suggest to create a new macro to define such intervals in the `math_notations.sty` so that the same notations are used throughout the doc.

Furthermore, this should be a lower case letter N instead of upper case, because it is not random.

Reply:
fixed